Fix boto_ec2 unused parameter "network_interfaces"

azelezni opened the pull request:
### What does this PR do?
Allow using `network_interfaces` parameter with boto_ec2.instance_present state module.

alexfirefly committed:
Fix unused network_interfaces parameter

When using the boto_ec2.instance_present state module,
network_interfaces parameter was not used.

rallytime commented:
Hi @azelezni and welcome!

There are some lint errors on this change: https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/21184/violations/file/salt/modules/boto_ec2.py/

The PY3 compatibility errors have suggested fixes in the "console" output link: https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/21184/console

Can you fix those?

alexfirefly committed:
Fix unused network_interfaces parameter

When using the boto_ec2.instance_present state module,
network_interfaces parameter was not used.

rallytime reviewed (approved):
Thank you @azelezni - this looks good to me.

rallytime commented:
@ryan-lane Do you want to look this one over?
